docs/source/environments/braintumor.rst

```diff
@@ -0,0 +1,163 @@
+.. _braintumor:
+
+
+Brain Tumor PDE
+================
+
+This documentation describes the Brain Tumor DPR (Diffusion-Proliferation-Radiation) Environment, which models the growth dynamics of glioblastomas–a fast-growing type of brain cancer–and their response to external beam radiation therapy (XRT). The model is governed by a partial differential equation (PDE) that includes three key terms: diffusion of tumor cells, proliferation, and radiation-induced cell death.
+
+While tumor growth naturally occurs in three spatial dimensions and time, the model assumes spherical symmetry, allowing the simulation to be reduced to a one-dimensional radial domain over time. To further ease computation, the PDE is nondimensionalized, rescaling spatial, temporal, and concentration variables into a unitless form. As a result, the tumor’s evolving state can be represented as a one-dimensional spatial array indexed by time.
```

Comment:
Change this sentence:

While tumor growth naturally occurs in three spatial dimensions and time, we capitalize on the spherical symmetry of tumors reducing the spatial dimension to a one-dimensional radial domain.